README.md

```diff
@@ -1,92 +1,102 @@
 # TICOI
+
 [![Language](https://img.shields.io/badge/python-3.10%2B-blue.svg?style=flat-square)](https://www.python.org/)
 [![Python test](https://github.com/ticoi/ticoi/actions/workflows/python-app.yml/badge.svg?branch=main)](https://github.com/ticoi/ticoi/actions/workflows/python-app.yml)
 [![License](https://img.shields.io/badge/license-GPLv3+-blue.svg?style=flat-square)](https://github.com/ticoi/ticoi/blob/main/LICENSE)
+[![Hatch project](https://img.shields.io/badge/%F0%9F%A5%9A-Hatch-4051b5.svg)](https://github.com/pypa/hatch)
+[![Ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff)
 
+TICOI is a post-processing method based on the temporal closure to fuse multi-temporal and multi-sensor velocity measurements,
```

Comment:
maybe you can just add one sentence to add the broader context. For a random person, what is "multi-sensor velocity measurements"?
Maybe "TICOI is a tool to postprocess surface velocity time series estimated from remote sensing (e.g., ice flow, landslides). The method is based on..."